gradle/libs.versions.toml

```diff
@@ -111,12 +109,10 @@ android-junit5 = { id = "de.mannodermaus.android-junit5", version.ref = "gradleP
 crashlytics = { id = "com.google.firebase.crashlytics", version.ref = "gradlePlugins-crashlytics" }
 detekt = { id = "io.gitlab.arturbosch.detekt", version.ref = "gradlePlugins-detekt" }
 googleServices = { id = "com.google.gms.google-services", version.ref = "gradlePlugins-gms" }
-hilt = { id = "dagger.hilt.android.plugin", version.ref = "gradlePlugins-hilt" }
```

Comment:
@hvisser not sure why these were removed? I am fixing this locally, but maybe it's a plugin bug?

Reply:
I'll take a look, thanks for the heads up.

Reply:
@rock3r I think that has to do with the manual mapping from plugin id to maven coordinates in the settings file, I've filed https://github.com/littlerobots/version-catalog-update-plugin/issues/11